promises.tex

```diff
@@ -415,7 +415,9 @@ \section{Using Promises}\label{s:promises-usage}
 but it's a lot more convenient if we allow users to provide it as a command-line argument
 when they run our script.
 To aid this,
-Node passes all of the command-line arguments in an array called \texttt{process.argv}.
+we'll use the property within Node called \texttt{process.argv}
+which returns the command line arguments as an array.
```

Comment:
```suggestion
which is an array containing all the command line arguments passed to the script.
```

Reply:
I'm not sure that I like the use of "returns" here as, to me, that implies that `process.argv` is a function rather than an object.

Reply:
Good call, thanks!